README.md

```diff
@@ -699,7 +699,21 @@ randomSubscriber.on('randomUpdate', (req) => {
 level with WebSockets over socket.io. `Sockend` makes `Responder`s and
 `Publisher`s available to the front-end and adhere to socket.io namespaces.
 It's the magic and the lost link for microservices. Without any configuration,
-you can expose APIs directly to the front-end.
+you can expose APIs directly to the front-end. 
+
+`Sockend` supports socket.io rooms. All you need to do is add `__rooms` attribute to 
```

Comment:
`_rooms` means that it will always be an array. Isn't this a limiting factor? I imagine quite a big portion of the time it will be a single room. how about we accept both arrays and a single room name?

Reply:
no problem. in that case we can support both __room and __rooms for clarity. but i kinda followed the same line as broadcasts:[], subscribesTo:[], etc..

Reply:
nice catch! except that broadcasts is 99% of the time an array.

for me the question is, should we have two separate values `__room` and `__rooms` or just have `__room` but it also accepts an array?

Reply:
i think that would be less intuitive...
imho:
__room: 'room1' - good (as a convience)
__rooms:['room1','room2'] - good (can also be used as __rooms:['room1'] of course)
__rooms: 'room1' - not so good, not intuitive

what do you say?

Reply:
you are right. let's go with `__room` and `__rooms` then.

Reply:
ok. on it. plus other fixes you requested...